src/components/EditImageModal/index.jsx

```diff
@@ -147,6 +148,11 @@ export default class EditImageModal extends React.Component {
 
     this.props.clearSearch(this.props.courseDetails);
     this.resetImageSelection();
+    this.modalWrapperRef.querySelector('.modal').dispatchEvent(new CustomEvent('closeModal',
```

Comment:
I assume this is related to your comment, `The EditImageModal sends a signal when the modal is closed now too.`

This is solely to handle the scrolling issue in Studio, right?

Is there a reason you have to query for `.modal` instead of dispatching the event on the modalWrapperRef itself? That's how we add the eventListener to the Modal.

Reply:
Yep, that's right. The `edit.js` code in the platform already had a reference to the `.modal` that it was listening to, so I made sure to emit the event where it is listening. Though, now looking at the platform code again, I think I can change it so that it is listening to the modal wrapper, and then I could dispatch this event from `this.modalWrapperRef` instead which would be cleaner.

Reply:
Ah, okay. That sounds good. I think that makes a little more sense conceptually too.